app/javascript/dashboard/routes/dashboard/settings/auditlogs/Index.vue

```diff
@@ -107,24 +107,28 @@ export default {
       const username = this.getAgentName(auditLogItem.username);
       const auditableType = auditLogItem.auditable_type.toLowerCase();
       const action = auditLogItem.action.toLowerCase();
+      const auditId = auditLogItem.auditable_id;
 
+      const startsWithVowel = /^[aeiou]/i.test(auditableType);
+      const articleKey = startsWithVowel ? 'ACTION_VOWEL' : 'ACTION_CONSONANT';
       const logActions = {
-        create: this.$t('AUDIT_LOGS.ACTION.ADD'),
-        destroy: this.$t('AUDIT_LOGS.ACTION.DELETE'),
-        update: this.$t('AUDIT_LOGS.ACTION.EDIT'),
-        sign_in: this.$t('AUDIT_LOGS.ACTION.SIGN_IN'),
-        sign_out: this.$t('AUDIT_LOGS.ACTION.SIGN_OUT'),
+        create: this.$t(`AUDIT_LOGS.${articleKey}.ADD`),
+        destroy: this.$t(`AUDIT_LOGS.${articleKey}.DELETE`),
+        update: this.$t(`AUDIT_LOGS.${articleKey}.EDIT`),
```

Comment:
@vishnu-narayanan This implementation is not correct. This creates a sentence as follows.

`Pranav created a new automationrule` not `Pranav created a new automation rule`.

Also on the language change this would out put a text like `Pranav creó un nuevo automationrule` which is not a proper translation. Also splitting a sentence like this makes it harder for translator to do a better job.

I would suggest to rewrite the implementation to the following. 

The translation file looks like this.
```json
{
  "AUTOMATION_RULE": {
    "ADD": "%{userName} create a new automation rule (%{id})"
  }
}
```

Index.vue changes to 

```js
const logActions = {
  "automationrule:add": this.$t('AUDIT_LOGS.AUTOMATION_RULE.ADD', { userName, id: auditableId  }),

}

// ...

return logActions[`${auditableType}:`${action}`]
```

Reply:
@pranavrajs Make sense. Will make the changes.